services/userService.js

```diff
@@ -2,6 +2,11 @@ const userDao = require("../models/userDao");
 const bcrypt = require("bcrypt");
 const jwt = require("jsonwebtoken");
 
+const checkEmail = async (email) => {
+  const user = await userDao.getByUserEmail(email);
+  return user;
+};
```

Comment:
```suggestion
const checkEmail = async (email) => {
  return userDao.getByUserEmail(email);
};
```
다른 추가적인 코드가 없으므로 service에서는 곧바로 리턴하여 처리합니다! (이때 위의 리뷰 내용에 따라 `true or false`가 return됩니다~